openj9Settings.mk

```diff
@@ -126,10 +126,21 @@ else
 	ADD_JVM_LIB_DIR_TO_LIBPATH:=export $(TEST_LIB_PATH);
 endif
 
+#######################################
+# In JDK18+, java.security.manager == null behaves as -Djava.security.manager=disallow.
+# In JDK17-, java.security.manager == null behaves as -Djava.security.manager=allow.
+# For OpenJ9 tests to work as expected, -Djava.security.manager=allow behaviour is
+# needed in JDK18+.
+#######################################
+ifeq (18,$(word 1,$(sort 18 $(JDK_VERSION))))
+	EXTRA_OPTIONS += -Djava.security.manager=allow
```

Comment:
Just gave a second thought, this might cause issue because the options don't have space in between. Maybe this is better:
```
EXTRA_OPTIONS := $(EXTRA_OPTIONS) -Djava.security.manager=allow
```

Reply:
As per the documentation (https://www.gnu.org/software/make/manual/html_node/Appending.html), both approaches are identical.